jina_commons/batching.py

```diff
@@ -29,7 +31,7 @@ def get_docs_batch_generator(
     :param docs: A document array.
     :param traversal_path: Specifies along which "axis" the document shall be traversed. (defaults to ['r'])
     :param batch_size: Size of each generated batch (except the last one, which might be smaller, default: 32)
-    :param needs_attr: Optionally, you can filter out docs which don't have this attribute
+    :param needs_attr: Optionally, you can filter out docs which don't have this content attribute
```

Comment:
this is not true, in theory. You could want this to extract different attributes, like `tags__brand` or whatever.

Reply:
That's true, I'll change back the description